server/container_create_linux.go

```diff
@@ -814,7 +814,9 @@ func addOCIBindMounts(ctx context.Context, mountLabel string, containerConfig *t
 		if dest == "" {
 			return nil, nil, fmt.Errorf("mount.ContainerPath is empty")
 		}
-
+		if dest == "/" {
```

Comment:
Isn't this only dangerous is source="/" also?

Reply:
it is less dangerous, but probably still worth reporting.  `$SOURCE/dev`, `$SOURCE/proc` and `$SOURCE/sys` can be propagated to the host

Reply:
I personally don't find it as risky, and openshift would report this by default with `oc debug node`, which diminishes the importance of the warning. I think we should only warn if source and dest are "/"